src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java

```diff
@@ -246,12 +249,13 @@ public boolean hasProject(String projectID, String organizationId) {
     public boolean canAdministerApplication() {
         // NOTE: user can administer application by default if running without authentication
         if (Auth0Connection.authDisabled()) return true;
+        if (app_metadata == null) return false;
+        if (app_metadata.getDatatoolsInfo() == null) 
```

Comment:
What happens here if `app_metadata.getDatatoolsInfo() == null` ?

Reply:
you're right I forgot a return false

Reply:
Done